<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bot de Pesca WoW 3.3.5a</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>751</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>271</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 251, 255);
border-bottom-right-radius:50px;
border-top-right-radius:50px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(0,0,0,200)</string>
    </property>
    <property name="text">
     <string>PAINEL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="input">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101 , 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string> New folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
	background-color:qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11, 131, 120, 219),stop:1 rgba(85, 98, 112,226));
	color:rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton#pushButton:hover{
	background-color: rgb(85, 170, 255);
}

QPushButton#pushButton:pressed{
		background-color: rgb(255, 0, 127);;
}
</string>
    </property>
    <property name="text">
     <string>CREATE </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posi_x">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101 , 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>LEFT_X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posi_y">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101 , 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>TOP_Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posi_a">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101 , 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>RIGHT_X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="posi_b">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(46, 82, 101 , 200);
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>BOTTOM_Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_2{
	background-color:qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11, 131, 120, 219),stop:1 rgba(85, 98, 112,226));
	color:rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton#pushButton_2:hover{
	background-color: rgb(85, 170, 255);
}

QPushButton#pushButton_2:pressed{
		background-color: rgb(255, 0, 127);;
}
</string>
    </property>
    <property name="text">
     <string>Show Screen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>380</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Social Media Circled</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_3{
	background-color:rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#pushButton_3:hover{
	color: rgba(131,96,53,225);
}


</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Social Media Circled</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_4:pressed{
	color:rgba(91,88,53,255);
}
QPushButton#pushButton_4{
	background-color:rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#pushButton_4:hover{
	color: rgba(131,96,53,225);
}

QPushButton#pushButton_4:pressed{
	color:rgba(91,88,53,255);
}</string>
    </property>
    <property name="text">
     <string>/C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>380</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Social Media Circled</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_5{
	background-color:rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#pushButton_5:hover{
	color: rgba(131,96,53,225);
}

QPushButton#pushButton_5:pressed{
	color:rgba(91,88,53,255);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>380</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Social Media Circled</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_6{
	background-color:rgba(0,0,0,0);
	color:rgba(85,98,112,255);
}
QPushButton#pushButton_6:hover{
	color: rgba(131,96,53,225);
}

QPushButton#pushButton_6:pressed{
	color:rgba(91,88,53,255);
}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>261</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/icon/mar.jpg);
border-bottom-left-radius:50px;
border-top-left-radius:50px;

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,80);
border-top-left-radius:50px;


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 210);
</string>
    </property>
    <property name="text">
     <string>FISHING BOT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 210);
</string>
    </property>
    <property name="text">
     <string>WotLK 3.3.5a </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="input_aviso">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>160</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 210);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_8{
	background-color:qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11, 131, 120, 219),stop:1 rgba(85, 98, 112,226));
	color:rgba(255,255,255,210);
	border-radius:10px;
}
QPushButton#pushButton_8:hover{
	background-color: rgb(85, 170, 255);
}

QPushButton#pushButton_8:pressed{
		background-color: rgb(255, 0, 127);;
}
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
color:rgba(0, 0, 0, 240);
padding-bottom:7px;</string>
    </property>
    <property name="text">
     <string>Press R Screen  Position</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="input_posi">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 210);
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>  y:      |     x: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>60</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_9{
	background-color:qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11, 131, 120, 219),stop:1 rgba(85, 98, 112,226));
	color:rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton#pushButton_9:hover{
	background-color: rgb(85, 170, 255);
}

QPushButton#pushButton_9:pressed{
		background-color: rgb(255, 0, 127);;
}
</string>
    </property>
    <property name="text">
     <string>Press  S Screen </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_7{
	background-color:qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11, 131, 120, 219),stop:1 rgba(85, 98, 112,226));
	color:rgba(255,255,255,210);
	border-radius:20px;
}
QPushButton#pushButton_7:hover{
	background-color: rgb(85, 170, 255);
}

QPushButton#pushButton_7:pressed{
		background-color: rgb(255, 0, 127);;
}
</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="stopwatch_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 210);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_10{
	background-color:qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11, 131, 120, 219),stop:1 rgba(85, 98, 112,226));
	color:rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton#pushButton:hover_10{
	background-color: rgb(85, 170, 255);
}

QPushButton#pushButton:pressed_10{
		background-color: rgb(255, 0, 127);;
}
</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_11{
	background-color:qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11, 131, 120, 219),stop:1 rgba(85, 98, 112,226));
	color:rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton#pushButton:hover_11{
	background-color: rgb(85, 170, 255);
}

QPushButton#pushButton:pressed_11{
		background-color: rgb(255, 0, 127);;
}
</string>
    </property>
    <property name="text">
     <string>PAUSE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_12{
	background-color:qlineargradient(spread:pad, x1:0, y1:0.505682,x2:1,y2:0.477,stop:0 rgba(11, 131, 120, 219),stop:1 rgba(85, 98, 112,226));
	color:rgba(255,255,255,210);
	border-radius:5px;
}
QPushButton#pushButton:hover_12{
	background-color: rgb(85, 170, 255);
}

QPushButton#pushButton:pressed_12{
		background-color: rgb(255, 0, 127);;
}
</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 210);
</string>
    </property>
    <property name="text">
     <string>Dev:Khufos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
